docs/Caching.md

```diff
@@ -0,0 +1,158 @@
+# Caching
+
+Web platform apps cache data at a few different levels in order to improve UI
+responsiveness. There are two primary types of content that are cached:
+
+1. REST API responses
+2. Static assets (files)
+
+## Overview
+
+![Cache diagram](https://user-images.githubusercontent.com/1885153/27892985-d75546fe-6256-11e7-8272-66252caa5c85.png)
+
+([from web platform diagrams](https://docs.google.com/presentation/d/1c8jf8UtGa81Ay4oqlbHfoP4Ile2VkiRVrCMVag6cRBQ/#slide=id.g1f921bb8da_0_0))
+
+## 1. API responses
+
+Currently the Meetup REST API does not provide a
```

Comment:
There's no particular reason why we're dependent upon caching headers to determine how mup-web should cache api responses, although we may decide it would be desirable to act upon them.